4-projects/modules/base_env/example_peering_project.tf

```diff
@@ -33,12 +33,31 @@ data "google_netblock_ip_ranges" "iap_forwarders" {
 module "peering_project" {
   source = "../single_project"
 
-  org_id          = local.org_id
-  billing_account = local.billing_account
-  folder_id       = local.env_folder_name
-  environment     = var.env
-  project_budget  = var.project_budget
-  project_prefix  = local.project_prefix
+  org_id                              = local.org_id
+  billing_account                     = local.billing_account
+  folder_id                           = local.env_folder_name
+  environment                         = var.env
+  project_budget                      = var.project_budget
+  project_prefix                      = local.project_prefix
+  enable_cloudbuild_deploy            = local.enable_cloudbuild_deploy
+  app_infra_pipeline_service_accounts = local.app_infra_pipeline_service_accounts
+
+  // The roles defined in "sa_roles" will be used to grant the necessary permissions
+  // to deploy the resources, a Compute Engine instance for each environment, defined
+  // in 5-app-infra step (5-app-infra/modules/env_base/main.tf).
+  // The roles are grouped by the repository name ("${var.business_code}-example-app") used to create the Cloud Build workspace
+  // (https://github.com/terraform-google-modules/terraform-google-bootstrap/tree/master/modules/tf_cloudbuild_workspace)
+  // in the 4-projects shared environment of each business unit.
+  // the repository name is the same key used for the app_infra_pipeline_service_accounts map and the
+  // roles will be granted to the service account with the same key.
```

Comment:
How about improve a little this comment block? Looks like the written is not super fluid here.
Example: instead of `...permissions to deploy the resources, a Compute Engine instance...`, how about `...permissions to deploy a Compute Engine instance...`

Also, maybe skipping a line between one idea and another could be easier to understand, instead of multiple non-breaking lines of text.